lib/src/view/game/analysis_screen.dart

```diff
@@ -0,0 +1,944 @@
+        : kEmptyWidget;
+  }
+}
+
+class _CevalLines extends ConsumerWidget {
+  const _CevalLines(this.ctrlProvider, {required this.isTablet});
+  final AnalysisCtrlProvider ctrlProvider;
+  final bool isTablet;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final analysisState = ref.watch(ctrlProvider);
+    final ceval =
+        ref.watch(engineEvaluationProvider(analysisState.evaluationContext));
+
+    final content = ceval != null
+        ? ceval.pvs
+            .map(
+              (pv) => _CevalLine(
+                ctrlProvider,
+                pv.sanMoves(ceval.currentPosition),
+                analysisState.currentNode.ply,
+                pv,
+              ),
+            )
+            .toList()
+        : (analysisState.currentNode.eval != null
+            ? analysisState.currentNode.eval!.pvs
+                .take(analysisState.numCevalLines)
+                .map(
+                  (pv) => _CevalLine(
+                    ctrlProvider,
+                    pv.sanMoves(analysisState.currentNode.position),
+                    analysisState.currentNode.ply,
+                    pv,
+                  ),
+                )
+                .toList()
+            : null);
+
+    return content != null
+        ? Padding(
+            padding: EdgeInsets.symmetric(
+              vertical: isTablet ? kTabletPadding : 4.0,
+              horizontal: isTablet ? kTabletPadding : 4.0,
+            ),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: content,
+            ),
+          )
+        : kEmptyWidget;
+  }
+}
+
+class _CevalLine extends ConsumerWidget {
+  const _CevalLine(
+    this.ctrlProvider,
+    this.moves,
+    this.initialPly,
+    this.pvData,
+  );
+
+  final AnalysisCtrlProvider ctrlProvider;
+  final List<String> moves;
+  final int initialPly;
+  final PvData pvData;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final builder = StringBuffer();
+    var ply = initialPly + 1;
+    moves.take(15).forEachIndexed((i, s) {
+      builder.write(
+        ply.isOdd
+            ? '${(ply / 2).ceil()}. $s '
+            : i == 0
+                ? '${(ply / 2).ceil()}... $s '
+                : '$s ',
+      );
+      ply += 1;
+    });
+
+    final (evalString, whiteBetter) = pvData.evalStringAndWinningSide;
+    return InkWell(
+      onTap: () => ref
+          .read(ctrlProvider.notifier)
+          .onUserMove(Move.fromUci(pvData.moves[0])!),
+      child: Padding(
+        padding: const EdgeInsets.all(2.0),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.start,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            Container(
+              padding: const EdgeInsets.all(2.0),
+              decoration: BoxDecoration(
+                color: whiteBetter != null && !whiteBetter
```

Comment:
I don't think we want to display the engine lines like the gauge in function of the winning side. I'd simply leave the normal background whatever the side.